src/compas/scene/sceneobject.py

```diff
@@ -150,6 +152,21 @@ def add(self, item, **kwargs):
         else:
             raise ValueError("Cannot add items to a scene object without a node.")
 
+    @property
+    def settings(self):
+        settings = {
+            "name": self.name,
```

Comment:
not now, but what to do with names and where to store them is something that still needs to be clarified (since the data item can have a name as well)

Reply:
agree, added a line in docstring about this